ci-operator/jobs/openshift/odo/openshift-odo-main-presubmits.yaml

```diff
@@ -0,0 +1,369 @@
+presubmits:
+  openshift/odo:
+  - agent: kubernetes
+    always_run: true
```

Comment:
I think always run should be `false` in our case. Correct me if I am wrong.

Reply:
done